woocommerce/woocommerce-gateway-stripe
# Update product page PRB when add-ons change

Fixes #2322 (at least for the official Product Add-Ons extension)

This fix triggers the same PRB update routine when Product Add-On
selections change that currently runs when the quantity is changed on
the product page.


## Testing instructions

- Follow the steps to reproduce the issue in #2322.
- Checkout this branch.
- Try the steps again and verify that the correct price is shown in the Google Pay or Apple Pay pop-up when a product add-on is chosen.

---

**Post merge**
